lib/config/presets/internal/group.ts

```diff
@@ -336,6 +336,16 @@ const staticGroups = {
       },
     ],
   },
+  react: {
+    description: 'Group React and corresponding `@types` packages together.',
+    packageRules: [
+      {
+        extends: 'monorepo:react',
```

Comment:
I don't think this will work, if it is based on `sourceUrl`, because it will AND together matchSourceUrls and matchPackageNames

Reply:
Oh interesting, I was under the imprsesion it would do an OR.

What would be the way to do that then? Essentially group together packages from the `"https://github.com/facebook/react"` `sourceURL` and packages matching `@types/react` and `@types/react-dom`?

Also we don't use it ourselves so I forgot about it, but the linked issue also refers to `@types/react-test-renderer`. I should probably include it to the PR for completeness once we reach the correct syntax (and as a reminder, we can't use `@types/react` as a prefix because there are a lot of them, mostly not matching packages from the React monorepo).